MobileClient/SignupScreen.js

```diff
@@ -0,0 +1,150 @@
+import React, { useState } from "react";
+import { StyleSheet, Text, View, Image, TextInput, TouchableOpacity } from "react-native";
+import Feather from 'react-native-vector-icons/Feather';
+
+export default function SignupScreen() {
+    Feather.loadFont();
+
+    const [entryP, setEntryP] = useState(true); //state for Password
```

Comment:
Should end be changed to viewPassword or something similar for clarity. Also I think it might be OK to have either eye effect both, but maybe this design is better? @Preetham-Syamala what do you think?

Reply:
I think it is more flexible this way. I will change the name to viewPassword.